mentat/config.py

```diff
@@ -35,22 +35,54 @@ class Config:
     # Model specific settings
     model: str = attr.field(
         default="gpt-4-1106-preview",
-        metadata={"auto_completions": list(known_models.keys())},
+        metadata={
+            "auto_completions": list(known_models.keys()),
+            "description": "Choose from one of the supported models: " + ", ".join(
+                list(known_models.keys())
+            ),
+        },
     )
     feature_selection_model: str = attr.field(
         default="gpt-4-1106-preview",
-        metadata={"auto_completions": list(known_models.keys())},
+        metadata={
+            "auto_completions": list(known_models.keys()),
+            "description": (
+                "The model used to prune the features before sending to the main model"
+                " for edits. Only used if llm_feature_filter is greater than 0."
+                " Possible models are: "
+                + ", ".join(list(known_models.keys()))
```

Comment:
Again, would prefer not to have model names in description.

Reply:
Curious why the preference against listing the supported models? Its a common approach to list the supported "items" - e.g. pandoc documentation lists the supported file formats, etc. It also helps when using the options on the commandline and selecting a model without having to run the app first or read the code.

Perhaps better formatting on the list of models? The list is exactly the same as the autocomplete. In the case of generated documentation there is typically no autocomplete (at most you might have some generated javascript with the list hidden and then displayed for html version of the documentation).

Reply:
For one, there are going to be a lot of available models and I think it would bloat the description way too much. I'm working on switching us to a new library right now that can support models from any provider easily, and I don't want 10 or 20 lines of models in the descriptions.

Reply:
I was going to suggest moving to a lib like llmware that had a lot of other support (like vetors, local models etc.). I would change the description from the hardcoded list (which will hopefully e gone with the change) to one referencing the lib at that point.